### PR TITLE
update project

### PR DESCRIPTION
**Pull request by asmallteapot**
new, improved, cleaned up for CI

### CODE REVIEW

**Comment by asmallteapot** [comment]
fixes #38

**Comment by asmallteapot** [comment]
fixes #43

File: lib/cocoapods-playgrounds/workspace.rb [hunk 1]
```diff
@@ -152,31 +152,23 @@ def generate_project
...
-        config.build_settings['ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME'] = 'LaunchImage'
-        config.build_settings['CODE_SIGN_IDENTITY[sdk=iphoneos*]'] = ''
-        config.build_settings['CODE_SIGNING_ALLOWED'] = 'NO'
-        config.build_settings['CODE_SIGNING_REQUIRED'] = 'NO'
-        config.build_settings['DEFINES_MODULE'] = 'NO'
-        config.build_settings['EMBEDDED_CONTENT_CONTAINS_SWIFT'] = 'NO'
+        update_build_settings(config)
       end
 
       # TODO: Should be at the root of the project
```
Comment (asmallteapot) [inline lib/cocoapods-playgrounds/workspace.rb]: TODO: fix in this PR

File: lib/cocoapods-playgrounds/workspace.rb [hunk 2]
```diff
@@ -152,31 +152,23 @@ def generate_project
...
-        end
-      end
+    def update_build_settings(config)
+      config.build_settings['ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME'] = 'LaunchImage'
+      config.build_settings['CODE_SIGN_IDENTITY[sdk=iphoneos*]'] = ''
+      config.build_settings['CODE_SIGNING_ALLOWED'] = 'NO'
+      config.build_settings['CODE_SIGNING_REQUIRED'] = 'NO'
+      config.build_settings['DEFINES_MODULE'] = 'NO'
+      config.build_settings['EMBEDDED_CONTENT_CONTAINS_SWIFT'] = 'NO'
+      # TODO: define swift version correctly
```
Comment (asmallteapot) [inline lib/cocoapods-playgrounds/workspace.rb]: TODO: fix in this PR

File: lib/cocoapods-playgrounds/workspace.rb [hunk 3]
```diff
@@ -23,9 +25,9 @@ def generate(install = true)
       Dir.chdir(target_dir) do
         setup_project(install)
 
-        generator = Pod::PlaygroundGenerator.new(@platform)
+        generator = Pod::PlaygroundGenerator.new(@platform, @names)
```
Comment (asmallteapot) [inline lib/cocoapods-playgrounds/workspace.rb]: TODO: this sends the requested spec names, which aren't necessarily importable

File: lib/cocoapods-playgrounds/workspace.rb (hunk 2 above)
Comment (asmallteapot) [reply lib/cocoapods-playgrounds/workspace.rb]: fixes #48

**Comment by asmallteapot** [comment]
fixes #52

**Comment by asmallteapot** [comment]
fixes #50

File: lib/cocoapods-playgrounds/workspace.rb (hunk 1 above)
Comment (asmallteapot) [reply lib/cocoapods-playgrounds/workspace.rb]: filed as #58

File: lib/cocoapods-playgrounds/workspace.rb (hunk 2 above)
Comment (asmallteapot) [reply lib/cocoapods-playgrounds/workspace.rb]: this works as a stopgap for common cases. filed #59 to adaptively pick the right version, allow a command line flag, and/or other appropriate fixes.

**Comment by Sajjon** [comment]
I solved it by downloading `Playground with Platform Choice.xctemplate` [from this Github repo](https://github.com/youngshook/XcodeFileTemplates) and placing it in `/Applications/Xcode.app/Contents/Developer/Library/Xcode/Templates/File Templates/Source`. Might be hacky but works (Xcode 10.1)

**Pull request merged**